recipes/Polycircles/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "polycircles" %}
+{% set version = "0.3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5e7a1872e88694bbaab98c97912e94babb60edb4be14f84c248dbe11960ac9e3
+
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```

Comment:
Can this be `noarch: python`?

Reply:
@scopatz Of course, I actually meant to do that, thank you!